fala/apps/adviser/tests/test_laa_laa_paginator.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from fala.apps.adviser.laa_laa_paginator import LaaLaaPaginator
+
+class LaaLaaPaginatorTest(unittest.TestCase):
```

Comment:
I would advocate for always using a Django testcase subclass. For stuff using a database, it will be more critical, and here we don't need stuff like the transactional behaviour of `django.test.TestCase`, but I think using [`django.test.SimpleTestCase`](https://docs.djangoproject.com/en/5.0/topics/testing/tools/#simpletestcase) is best here

I believe the django test runner would still find and run `unittest.TestCase` tests just fine, but using `django.test.SimpleTestCase` gives us some useful utilities like setup and teardown hooks (although for these tests we don't need those, admittedly) and -- most importanly -- consistency with other applications where we have a db and will definitely want to use Django subclasses of TestCase.